src/common/config.h.in

```diff
@@ -1,6 +1,7 @@
 #cmakedefine MENDER_BUFSIZE @MENDER_BUFSIZE@
 #cmakedefine MENDER_LOG_BOOST
 #cmakedefine MENDER_USE_NLOHMANN_JSON @MENDER_USE_NLOHMANN_JSON@
+#cmakedefine01 MENDER_USE_YAML_CPP
```

Comment:
Why treat this one differently?

Reply:
But it means that `#ifndef` won't work, which it does for the others. IMHO, we should keep them the same, IOW:

* `#cmakedefine` - Use `#ifdef` and `#ifndef`.
* `#cmakedefine01` - Use `#if` and `#if !`.

Reply:
So either is fine with me, as long as they are the same.

Reply:
Then, I'm just going to to `cmakedefine`